### ISSUE TITLE
Change arrangement of items within plots and characters?

### ISSUE THREAD

**Issue by sprite-1** [olivierkes/manuskript#560]
There doesn't seem to be an easy way to arrange plot points and characters. They aren't sorted alphabetically and dragging around doesn't seem to work. They're permanently sorted by date of addition which I feel may make things confusing if I want to insert things in between stuff or put the more relevant characters higher in the list.

Is this going to be a thing in the future or is it too difficult to implement? Thanks as always for the awesome software!

**Comment by n7jordanshepard** [comment]
New user here. 

I've taken a look at TheJackMonster's added post but still can't figure out if there is a way to sort plot points and characters alphabetically. Any ideas?

**Comment by TheJackiMonster** [comment]
No, I think sorting and filtering both doesn't work yet in those views.
